src/contract.js

```diff
@@ -55,7 +55,7 @@ function ContractFactory(contractABI, extend) {
                     to: self.address,
                   });
 
-                methodTxObject.data = encodeMethodReadable(methodObject, methodArgs)
```

Comment:
yeah, doesn't matter much, main thinking was anything interfacing with this should also be able to consume stuff from web3 eth-js etc, but the only thing that consumes this is a txparams to uri function